src/olympia/migrations/920-webext-permissions-file-fk-is-unique.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `webext_permissions` DROP FOREIGN KEY `webext_permissions_file`;
+
+-- This will fail locally as file.id is by default signed, but unsigned on prod, etc.
+ALTER TABLE `webext_permissions` MODIFY `file_id` int(11) unsigned UNIQUE NOT NULL;
```

Comment:
I think you may be able to avoid that by just doing `ALTER TABLE `webext_permission ADD UNIQUE (`file_id`);` (and not removing/re-adding the FK constraint)